rpcs3/Emu/RSX/RSXThread.h

```diff
@@ -783,6 +783,7 @@ namespace rsx
 		virtual u64 timestamp();
 		virtual bool on_access_violation(u32 /*address*/, bool /*is_writing*/) { return false; }
 		virtual void on_invalidate_memory_range(const address_range & /*range*/, rsx::invalidation_cause) {}
+		void invalidate_memory_range(const address_range & /*range*/, rsx::invalidation_cause) {}
```

Comment:
is there a special reason for the space between adress_range and & ?

Reply:
I personally prefer it without the space, but it's just a copy pasta of the above line.